test/byContract/IexecERC20/IexecERC20.test.ts

```diff
@@ -38,7 +37,8 @@ describe('ERC20', async () => {
         // Setup current test accounts from some arbitrary iExec accounts
         ({ requester: holder, beneficiary: recipient, anyone } = accounts);
         spender = recipient;
-        zeroAddressSigner = await ethers.getImpersonatedSigner(zeroAddress);
+        zeroAddressSigner = await ethers.getImpersonatedSigner(ZeroAddress);
+        await fundTheZeroAddress();
```

Comment:
Do we need it?

Reply:
![image](https://github.com/user-attachments/assets/ebfe19d7-733d-41f5-84ee-cdb44c353a77)

Reply:
For token chain right thanks :+1: